extensions/ql-vscode/src/model-editor/methods-usage/methods-usage-data-provider.ts

```diff
@@ -71,6 +71,7 @@ export class MethodsUsageDataProvider
         iconPath: new ThemeIcon("symbol-method"),
       };
     } else {
+      const method = this.getParent(item);
```

Comment:
This works, but I don't love that we iterate through all external API usages to find the right one. I think this is fine for this PR but then I have a suggestion for avoiding the need to call `getParent` here, and also making `getParent` itself faster. I'll open as a separate PR on top of this one.

Reply:
Agreed this is not great long term. I'm guessing that we'll need to start storing the relationship soon so we don't have to do this.

Reply:
I've opened https://github.com/github/vscode-codeql/pull/2774 which attempts to add this relationship

Reply:
Cool. I'll go ahead and merge this and we can discuss there.